resolwe_bio/docker_images/modeling/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM docker.io/resolwebio/base:ubuntu-16.04
+
+MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/resolwe-bio
+
+COPY requirements.txt .
```

Comment:
Since this image is derived from `resolwebio/base`, Python modules can also be added like [this](https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/docker_images/rnaseq/packages-python3.txt). Then you can remove the last two lines from your `Dockerfile`.